src/components/checkbox/Checkbox/checkbox.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../../styles/theme";
+
+.checkbox-wrapper {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  .text {
+    margin-left: $spacing_0_8;
+    font-size: $font_size_1_4;
+    font-weight: $font_medium;
+    color: &color_black_37;
+  }
+
+  & > input {
+    -webkit-appearance: none;
+    appearance: none;
+
+    width: 2rem;
+    height: 2rem;
+    border-radius: 0.5rem;
+    border: $color_gray_92989b 0.2rem solid;
+
+    &:hover {
+      border: $color_secondary_9749fa 0.2rem solid;
+    }
+
+    &.checked {
+      border: 0;
+      background-color: $color_secondary_9749fa;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &.checked::before {
```

Comment:
There is a difference between the styling of the check mark between the one here and the one in Figma.

I tried this instead, and it gets a bit closer to the design, but the ends of the check mark are still not rounded.

 &.checked::before {
      content: "";
      width: 0.4rem;
      height: 0.9rem;
      border: solid $color_white_ff;
      border-width: 0 0.2rem 0.2rem 0;
      transform: rotate(45deg);
      margin-bottom: 0.3rem;
    }

@vasilen-7DIGIT any idea ?